lib/mappers/qdc_mapper.py

```diff
@@ -31,10 +31,22 @@ def map_data_provider(self):
                                      getprop(self.provider_data, "publisher")})
 
     # sourceResource mapping
+    #
+    # FIXME: these method names should be changed to use verbs.
+    # E.g. `assign_sr_prop_from_same' and `assign_sr_prop_from_many'
```

Comment:
It doesn't seem to me that worrying about things at this level is useful for a legacy project we're planning on replacing.